rllib/evaluation/rollout_worker.py

```diff
@@ -283,6 +285,7 @@ def __init__(
         policies_to_train=DEPRECATED_VALUE,
         extra_python_environs=DEPRECATED_VALUE,
         policy=DEPRECATED_VALUE,
+        tf_session_creator=DEPRECATED_VALUE,  # Use config.tf_session_options instead.
```

Comment:
Simplified rollout_worker by making session_creator be inferred automatically from (the anyways available) config.